app/models/occupation_standard.rb

```diff
@@ -16,9 +16,24 @@ class OccupationStandard < ApplicationRecord
 
   delegate :title, to: :organization, prefix: true
   delegate :title, to: :occupation, prefix: true
-  delegate :title, to: :industry, prefix: true
+  delegate :title, to: :industry, prefix: true, allow_nil: true
   delegate :name, to: :creator, prefix: true
 
+  class << self
+    def filter_collection(options={})
```

Comment:
I know this is technically just filtering right now, but not knowing how it'll evolved, I'd be inclined to just make this another `search` for consistency.